docs/release-notes/14-6-0/README.md

```diff
@@ -8,20 +8,61 @@ release_date: 2024-09-26
 
 # OpenProject 14.6.0
 
-Release date: 2024-09-26
+Release date: 2024-10-09
 
-We released OpenProject [OpenProject 14.6.0](https://community.openproject.org/versions/2111).
-The release contains several bug fixes and we recommend updating to the newest version.
-In these Release Notes, we will give an overview of important feature changes.
-At the end, you will find a complete list of all changes and bug fixes.
+We released OpenProject [OpenProject 14.6.0](https://community.openproject.org/versions/2111). The release contains several bug fixes and we recommend updating to the newest version. 
+
+In these Release Notes, we will give an overview of important feature changes. At the end, you will find a complete list of all changes and bug fixes.
 
 ## Important feature changes
 
-<!-- Inform about the major features in this section -->
+### Updated progress reporting with most flexible options
+
+In response to valuable user feedback, OpenProject 14.6 brings important changes to progress reporting. We greatly appreciate the input from our Community, which helped shape these updates. Based on your feedback, the % Complete field is now manually editable again in work-based mode (like it has been in the versions before 14.0), removing the previous requirement to input Work or Remaining Work. This gives users more flexibility in tracking project progress.
+
+But we did not only revise the changes we made with 14.0, we also added new options to enhance progress tracking and make it most flexible for our users. This is why, additionally, administrators now have new settings for progress calculation and % Complete hierarchy totals:
+
+![Screenshot of the new admin page for progress tracking with OpenProject 14.6](progress-tracking-admin-options.png)
+
+If you use our progress tracking, we advise reading [this detailed article about progress reporting with OpenProject 14.6](https://www.openproject.org/blog/updates-to-progress-tracking-in-14-6-based-on-user-feedback/) from our product team.
+
+### Meetings: Receive an update notification with reload option
+
+If you have not used our Meetings module, you should consider starting it now. We updated dynamic meetings so that you now receive a notification whenever someone else made changes to the meeting you're looking at. Click on the button included in the notification to reload the page. This is a particularly great improvement for teams who work simultaneously in meetings.
```

Comment:
@corinnaguenther maybe make a small mention that the changes which were not yet saved will be lost when the reset button is clicked and advise users to save them elsewhere if they are extensive